PanKanapka/clients/forms.py

```diff
@@ -2,16 +2,34 @@
 from django.contrib.auth.forms import ReadOnlyPasswordHashField
 from django.contrib.auth.forms import UserChangeForm
 
+from django.core.exceptions import ValidationError
 from .models import User
 
+EMPTY_ELEMENT = "Pole %s nie może być puste"
 
 class RegisterForm(forms.ModelForm):
-    password = forms.CharField(widget=forms.PasswordInput)
-    password2 = forms.CharField(label='Potwierdź hasło', widget=forms.PasswordInput)
+
+    error_messages = {
+        'password_mismatch': "Hasła nie są identyczne",
+    }
+
+    password1 = forms.CharField(widget=forms.PasswordInput())
+    password2 = forms.CharField(label='Potwierdź hasło', widget=forms.PasswordInput())
 
     class Meta:
         model = User
-        fields = ('email',)
+        fields = ('email','name', 'surname')
+        widgets = {
+                'email': forms.fields.EmailInput(attrs={
```

Comment:
Może warto byłoby dodać jako pole przy rejestracji "Firma", to będzie konieczne do składania zamówień.